translate/src/modules/search/components/SearchPanel.tsx

```diff
@@ -38,7 +38,7 @@ const SearchOption = ({
         onToggle();
       }}
     >
-      <i className='fa fa-w'></i>
```

Comment:
Was this meant to be `fa-fw` (fixed-width)? In any case, the menu seems to work fine without it.